src/Javis.jl

```diff
@@ -219,16 +219,25 @@ The current settings of an [`Action`](@ref) which are saved in `action.current_s
 - `mul_opacity::Float64`: the current multiplier for opacity.
     The actual opacity is then: `mul_opacity * opacity`
 - `fontsize::Float64` the current font size
+- `scale::Tuple{Float64, Float64}`: the current scale
+- `mul_scale::Float64`: the current multiplier for scale.
+    The actual scale is then: `mul_scale * scale`
 """
 mutable struct ActionSetting
     line_width::Float64
     mul_line_width::Float64 # the multiplier of line width is between 0 and 1
     opacity::Float64
     mul_opacity::Float64 # the multiplier of opacity is between 0 and 1
     fontsize::Float64
+    # scale has current scale, scale and scale multiplier
+    # such that scale*mul_scale is the desired scale
+    # the scale change needs to be computed using current_scale
```

Comment:
I am a little confused by these comments. What do you mean by "scale has current_scale"? I am overall just confused about what is stored where based on these comments. Could you perhaps clarify them or explain them to me in a different way?

Reply:
Scale saves the current scaling factor whereas the other two combined give the desired new scaling.

Reply:
Changed the comment

Reply:
Hey, I think somehow this comment got outdated. You still talk about scale although it is no longer used.

Reply:
Done